scripts/transition_metrics.py

```diff
@@ -80,15 +117,57 @@ def rx_commission_decommission(filename, non_lwr):
                 '_exit')).fillna(0)
     else:
         simulation_data = c.fillna(0)
-
-    simulation_data = simulation_data.set_index('Time')
-    simulation_data['lwr_total'] = simulation_data.drop(
-        non_lwr, axis=1).sum(
-        axis=1)
-    simulation_data['lwr_total'] = simulation_data['lwr_total'].cumsum()
+        simulation_data = simulation_data.add_suffix('_enter')
+        for column in simulation_data.columns:
+            simulation_data[(column[:-5] + 'exit')] = 0.0
+    simulation_data['lwr_total'] = (
+        simulation_data['lwr_enter'] +
+        simulation_data['lwr_exit']).cumsum()
     return simulation_data.reset_index()
 
 
+def prototype_totals(outfile, nonlwr, prototypes):
+    '''
+    This function performs the tm.rx_commissions_decommission
+    function on a provided database. Then the total number of
+    each prototype deployed at a given time is calculated and
+    added to the dataframe
+
+    Parameters:
+    -----------
+    out_file: str
+        name of database
+    nonlwr: list of str
```

Comment:
Suggestion for clarity
```suggestion
    nonlwr_prototype_names: list of str
```